interactions/client/context.py

```diff
@@ -67,16 +66,48 @@ def __attrs_post_init__(self) -> None:
         if self.member and self.guild_id:
             self.member._extras["guild_id"] = self.guild_id
 
-        self.author = self.member
-
         if self.user is None:
             self.user = self.member.user if self.member else None
 
-        if self.guild is None and self.guild_id is not None:
-            self.guild = self._client.cache[Guild].get(self.guild_id, MISSING)
+        if not self.member.user and self.user:
+            self.member.user = self.user
+
+    @property
+    def created_at(self) -> datetime:
+        """
+        .. versionadded:: 4.4.0
 
-        if self.channel is None:
-            self.channel = self._client.cache[Channel].get(self.channel_id, MISSING)
+        Returns when the interaction was created.
+        """
+        return self.id.timestamp
+
+    @property
+    def author(self) -> Member:
+        """
+        Returns the author/member that invoked the interaction.
+        """
+        return self.member
+
+    @property
+    def channel(self) -> Optional[Channel]:
+        """
+        .. versionchanged:: 4.4.0
+            Channel now returns ``None`` instead of ``MISSING`` if it is not found to avoid confusion
+
+        Returns the current channel.
+        """
+        return self._client.cache[Channel].get(self.channel_id, None)
+
+    @property
+    def guild(self) -> Optional[Guild]:
+        """
```

Comment:
Docstrings should maybe show the difference between `ctx.guild` and `ctx.get_guild()`. Same goes for author and channel?

Reply:
man that is mean, I wanted you to make it so I could commit it xD

give me a moment

Reply:
hahaha I knew you wanted that, sorry I have assignments to give back xD